recovery/elliptics_recovery/recovery.py

```diff
@@ -55,6 +55,25 @@ def cleanup(path):
                 os.unlink(file_name)
 
 
+def get_routes(ctx):
+    log.debug('Requesting routes')
+
+    log.debug("Creating logger")
+    elog = elliptics.Logger(ctx.log_file, int(ctx.log_level))
+
+    log.debug("Creating node")
+    node = elliptics_create_node(address=ctx.address,
```

Comment:
Could those processes grab route table from own nodes?

Or route information is used to split requests among processes?